src/App.jsx

```diff
@@ -1,17 +1,28 @@
 import { useState } from 'react'
+import {useEffect} from 'react'
```

Comment:
you can import `useEffect` from `react` on line one rather than having multiple lines